src/steam_network/protocol/protobuf_socket_handler.py

```diff
@@ -452,21 +408,23 @@ async def _send_recv_common(self, msg: Message, job_id: int, callback : Callable
                 logger.info("[Out] %s (%dB)", repr(emsg), len(data))
             await self._socket.send(data)
         except Exception as e:
-            logger.error("Unexpected error sending the data.")
-            if (info is not None):
+            logger.error(f"Unexpected error sending the data: {e}")
+            logger.debug(traceback.format_exc())
```

Comment:
alternatively: `logger.exception`

Reply:
Thanks, I always forget that this method exists